instrumentation/opentelemetry-api-beta/src/test/groovy/MeterTest.groovy

```diff
@@ -96,34 +92,34 @@ class MeterTest extends AgentTestRunner {
     def instrument = meter."$builderMethod"("test")
       .setDescription("d")
       .setUnit("u")
-      .setConstantLabels(["m": "n", "o": "p"])
+      .setConstantLabels(Labels.of("m", "n", "o", "p"))
       .build()
     if (bind) {
-      instrument = instrument.bind()
+      instrument = instrument.bind(Labels.empty())
     }
     if (bind) {
       instrument.record(value1)
       instrument.record(value2)
     } else {
-      instrument.record(value1, "q", "r")
-      instrument.record(value2, "q", "r")
+      instrument.record(value1, Labels.of("q", "r"))
+      instrument.record(value2, Labels.of("q", "r"))
     }
 
     then:
-    def metricData = metricData(instrumentationName, "test")
+    def metricData = findMetric(OpenTelemetrySdk.getMeterProvider().getMetricProducer().getAllMetrics(), instrumentationName, "test")
     metricData != null
     metricData.descriptor.description == "d"
     metricData.descriptor.unit == "u"
     metricData.descriptor.type == SUMMARY
-    metricData.descriptor.constantLabels == ["m": "n", "o": "p"]
+    metricData.descriptor.constantLabels == io.opentelemetry.common.Labels.of("m", "n", "o", "p")
     metricData.instrumentationLibraryInfo.name == instrumentationName
     metricData.instrumentationLibraryInfo.version == "1.2.3"
     metricData.points.size() == 1
     def point = metricData.points.iterator().next()
     if (bind) {
-      point.labels == ["w": "x", "y": "z"]
+      point.labels == io.opentelemetry.common.Labels.of("w", "x", "y", "z")
```

Comment:
import?

Reply:
We already have an import for `unshaded.io.opentelemetry.common.Labels`, which is used for creating instruments.

Reply:
oh I see. carry on, then.